openwrt-openwisp-monitoring/files/netjson-monitoring

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+#
+# NetJSON Monitoring
+
+show_help() {
+	printf "Usage:\n"
+	printf "  netjson-monitoring [OPTIONS...]\n"
+	printf "\n"
+	printf "Use Netjson Monitoring to get current device metrics data.\n"
+	printf "\n"
+	printf "Options:\n"
+	printf "  -h, --help\t\t\t\t: Show this help text\n"
+	printf "  -v, --version\t\t\t\t: Shows version of the agent.\n"
+	printf "  --dump <interfaces to be monitored>\t: Dump device metrics data.\n"
```

Comment:
I think this point needs more explanation.
It's not really a list of interface to be monitored but a list of interfaces from which traffic data is extracted.

I think it should be something like:

--dump "<interfaces>"

And then below something like:

"<interfaces>" a space separated list of interfaces from which traffic statistics will be collected (tx_bytes, rx_bytes, etc).
Passing an asterisk here will extract traffic statistics from all interfaces.